### PR TITLE
Replaces the slime consoles with glass tables

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

requested by @OakBoscage
image

#### Changelog

:cl:  
tweak: Removed the slime consoles from xenobio, and replaced them with glass tables
/:cl:

### CODE REVIEW

**Comment by Partheo** [comment]
THANK YOU

**Comment by ghost** [comment]
why?

**Comment by Oakboscage** [comment]
Speedmerge please.

**Comment by Kmc2000** [comment]
approved

**Comment by ghost** [comment]
atleast remove the freezer room instead of doing a half assed job.

**Comment by ghost** [comment]
also you might as well remove cerulean blueprints now

**Comment by Kmc2000** [comment]
Ok nich asked me to go into detail


reasons this is good:


Reasons removing the consoles is dumb:

Youre making a job harder and shittier for no reason.


Give an actual reason as to why this is helpful

**Comment by X-TheDark** [comment]
Personally, I approve. This is a convenience and for people that cannot into manually pulling/feeding slimes (otherwise known as scrubs) like the GLOЯIOUS old days. /s

**TL;DR : Slime breeding without consoles can seem like a chore due to how much time it takes to move/kill/feed slimes manually, but, as useful as slime console is for making the whole process go A LOT faster, it also removes all inherent danger, which to me is a big part of Xenobiology. There are good reasons to have and not have them, depending on your perspective.**

**Why I'm AGAINST removing slime consoles:**

1. After slime splits, the fucker needs to be moved around, which takes quite a bit of time, especially if you want to keep all 4 slimes or something;
2. The moving around itself is finicky as fuck, because slimes detach from your passive grab/attack random shits in your vicinity/you;
3. Killing slimes manually takes quite a bit of time. Moving slimes to freezing chamber manually would be slower than just fire extinguishing it;
4. Having more than 1 xenobio working on slime breeding becomes actually doable and not dick-punchingly aggravating due to different aggro/friend stances for slimes;

The above points result in you spending about 50% of your time feeding (if that) with the other being split between moving slimes and killing them. This becomes especially egregious when you have all slime pens full. Walking between them and managing slimes (kill/move) manually takes so much time, you'd need to be able to bear with it to enjoy the process.


**Why I'm FOR removing slime consoles:**

1. Slime console trivializes slime killing;
2. Slime console trivializes slime moving; 
3. Slime console trivializes slime feeding;

Some of the most fun rounds I had were when some rando comes into my lab and !FUN! ensues for all people involved. In addition, nothing beats the uncertainty of being killed by your un-faithful roundstart slimes (after a bit of feeding, it usually becomes pretty safe). Fire extinguishing the slime manually always bore some legitimate danger, which, personally, was the fun part.

So, this raises the question : Is it possible to make slime breeding not shit effortwise (spending a ton of time doing useless micromanagement) while still retaining all the dangers and lethality of the old "manually operated" system?

**Comment by Partheo** [comment]
I think it should be shit effort wise. Xenobiologists are already some of the most annoying crewmembers for admins and the players, right behind geneticists with full powers. Every round we get a slew of sentient creatures and golems which should be, in my opinion, rare.

**Comment by Partheo** [comment]
Besides, everything was fine before it was added.

**Comment by X-TheDark** [comment]
`Besides, everything was fine before it was added.`

Here's the problem with that reasoning : Slime management (just the moving part alone) became objectively better after this feature was added.

Breeding slimes became boring, but it became faster and easier to manage (not talking about safety here, it just "feels" better with the console).

**Comment by ShadowDeath6** [comment]
reasons why this is bad: the entire pr

don't remove quality of life while adding nothing, adding static dumb difficulty to a job with little to no added depth is the opposite of improving it 

refer to https://github.com/yogstation13/yogstation/pull/1797

**Comment by Partheo** [comment]
Strongly disagree with Shadow. It's not adding nothing. Here's what it adds:

Risk.
Risk is the reason we have a singularity engine instead of a simple safer alternative. Risk is the reason the AI can go rogue. Risk is the reason Lavaland exists.
It's a core element that's fundamental to SS13. Adding features that remove risk and make interesting jobs like xenobiology too easy are not only bad, they betray the very basis of what this game is based on.

It's the same reason you wouldn't add an "Engineering Management Console" that turns the emitters on, activates the field, and places the generator on at range without having to go into space.

It's the same reason you wouldn't have a "Atmos Management Console" that moves canisters for you, allows you to set pumps at range, and replaces pipes and devices, just like build mode.

The list goes on.

If anything, the console removes depth- adding it back is just going to make the job tricky and dangerous again, which is good.

I think the real reason people are arguing is that it makes the game harder and is a straight nerf to xenobiologists. Don't be pissed that your job is getting nerfed if it's going against the very fundamental principles of what SS13 is all about.

That being said, being able to research the board is not a bad idea.

**Comment by matskuman5** [comment]
I support this only if the computer is made researchable

**Comment by Ratvar** [comment]
I support researchable computers if they are somehow gated, so xenobio cannot rush them with superparts like they always do and then fuck off. 
I think that console removal won't fix the problem, just will make it more of a hussle. Still can make 4 cores out of one slime, still can make 4 monkeys out once corpse so you never need gray slimes - aka updates to monkey/slime gibbers that make one golem into four golems, removing any need to breed same types.

**Comment by ghost** [comment]
lmao there's no risk with slime they're usually docile as fuck unless you dont feed them, (but your job is literaly feeding them). this isn't about risk this is about TG/Yog rounds lastin for like 40-60 minutes and thus  any real progress is difficult to make without consoles.


Also the entire engine argument is shit because the singulo is so easy and boring that TG replaced it. Also engineering spends about 10 minutes on the engine while xenobio spends all round on it.

You're clearly not realizing this isn't about risk, but it's about the job being fucking dull without the console.

**Comment by KrabSpider** [comment]
Slimes have no relationships at roundstart. Feeding them 2 monkeys makes them form a relationship.

**Comment by ToGWtF** [comment]
Few suggestions


Maybe lower the amount of slimes the consoles can pick up at a time? Right now its 4 at once. Makes mass killing grays easy which gives a BIG head start to xeno work.  Maybe lower it to 2, or even 1. Even if its moving just 1 slime at a time, the console camera moves a lot faster than a person can

Don't let the consoles drop monkeys. Feeding by hand adds back that risk and takes more time than double-clicking the monkey button. Even if they use the disposals to negate the risk, its going to take a lot more time.



Reminder, I ported the upgraded temperature guns designed to kill slimes that were never once used because we ported xenobiology consoles right after and can re-make my xenobiology firing pins 👍

**Comment by Amelia0010** [comment]
Only 1 person upvoted, 4 downvoted. Something something democracy

**Comment by Partheo** [comment]
:(

Six people supported it in the comments...

**Comment by Amelia0010** [comment]
learn how to vote :^)
